tests/storage/snapshot.go

```diff
@@ -1306,6 +1306,49 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 
 				Expect(snapshot.Status.CreationTime).To(BeNil())
 			})
+
+			It("vmsnapshot should update error if vmsnapshotcontent is unready to use and error", func() {
+				vm = tests.StartVMAndExpectRunning(virtClient, vm)
+				// Delete DV and wait pvc get deletionTimestamp
+				// when pvc is deleting snapshot is not possible
```

Comment:
Do we need the PVC to be there when we create the snapshot? What if the PVC is already deleted?

Reply:
pvc cant be deleted as long as it it mounted in the vm, once the vm is down it can and will be deleted. We need the pvc to be in deleting phase cause if it is already deleted then we have no problem since there will not be a volume snapshot done for it.